addons/arrays/test_shuffle.sqf

```diff
@@ -21,18 +21,17 @@ TEST_OP(count _result,==,count _original,_fn);
     TEST_OP(_x,in,_original,_fn);
 } forEach _result;
 
-// Test depecated version.
 _original = [1, 2, 3];
 _result = _original call CBA_fnc_shuffle;
```

Comment:
No. Update the way the function is called and then do the same two checks that the original unit test performs.